### ISSUE TITLE
elaborate on use-case/file format

### ISSUE THREAD

**Issue by raylu** [yining/coverage2lcov#1]
I noticed that `coverage lcov` is quite slow because it needs to parse all the source code into ASTs to figure out the number of statements (`LF`)

I thought this would solve my problem, but it appears it tries to parse `.coverage` as some kind of text-based line-delimited format. AFAICT, coveragepy switched from marshal to pickle to JSON to sqlite. I don't understand what format this is parsing

**Comment by yining** [comment]
hi @raylu 

this project was to address "a very specific problem in one of my workflows" where I had to use an *old* version of `coveragepy` that does not support outputting `lcov` format. It does by parsing the `.coverage` file generated by `coveragepy` and generating in the corresponding `lcov` format.

If you are using a relative new `coveragepy`, it's most likely not what you have been looking for. I am sorry for the confusion.

**Comment by raylu** [comment]
right, I saw that. and I checked out coveragepy 6.2, but that still generates a sqlite `.coverage` file, not something like https://github.com/yining/coverage2lcov/blob/84a9ad85acf4a57a8ff4474fa30e676e03bb2f01/src/lib.rs#L10

**Comment by yining** [comment]
IIRC, the version I used was `4.5.x`... anyway, try `coverage report --show-missing`
